packages/starlight-typedoc/tests/unit/typedoc.test.ts

```diff
@@ -165,13 +185,16 @@ test('should output index with correct module path', async () => {
   ).toBe(true)
 })
 
-function generateTestTypeDoc(options: Parameters<typeof generateTypeDoc>[0]) {
+function generateTestTypeDoc(
+  options: Parameters<typeof generateTypeDoc>[0],
+  config: AstroConfig = starlightTypeDocAstroConfig as AstroConfig,
```

Comment:
```suggestion
  config: Partial<AstroConfig> = starlightTypeDocAstroConfig,
```

As it's only for tests, let's move the cast in this function so it avoids the need to do it in tests.